README.md

```diff
@@ -9,3 +9,76 @@ This package is available on PyPI using:
 ```bash
 pip install qiskit-sphinx-theme
 ```
+
+## Configure Left Sidebar
+
+To keep UX/UI similar across different Qiskit packages we strongly encourage the following structure for you sidebar, which can be set in the toctree of your `index.rst`:
+
+```
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+
+   Overview <overview>
+   Getting Started <getting_started>
+   Tutorials <tutorials>
+   How-to Guides <how_to>
+   API Reference <apidocs>
+   Explanations <explanations>
+   Release Notes <release_notes>
+   GitHub <https://github.com/Qiskit/your-repo>
+```
```

Comment:
In #78 @HuangJunye and I discussed the structure of the four documentation type folders (`tutorials`, `how_to`, `apidocs` and `explanations`) and decided it was best to create an `index.rst` inside each one of these folders instead of having, for example, `tutorials.rst` and `how_to.rst` in the `docs` folder.

That would mean changing `<tutorials>` $\rightarrow$ `<tutorials/index>` and so on.